barretenberg/cpp/src/barretenberg/stdlib/eccvm_verifier/eccvm_recursive_verifier.test.cpp

```diff
@@ -84,7 +84,7 @@ template <typename RecursiveFlavor> class ECCVMRecursiveTests : public ::testing
         OuterBuilder outer_circuit;
         RecursiveVerifier verifier{ &outer_circuit, verification_key };
         verifier.verify_proof(proof);
-        info("Recursive Verifier: num gates = ", outer_circuit.num_gates);
+        info("Recursive Verifier: num gates = ", outer_circuit.get_estimated_num_finalized_gates());
```

Comment:
can you print out that this number is an estimate? or finalize the circuit and print out the actual finalized gate count

Reply:
it's quite expensive to actually finalise the circuit - is it important for this number to be 100% accurate?

Reply:
Shouldn't be too expensive right? Its fine to just keep it as is but say that its num estimated finalized gates instead of just num gates though.